### DISCUSSION TITLE
Command 'Reload Vibrancy' resulted in an error

### DISCUSSION THREAD

**Discussion by michaelberge8** [illixion/vscode-vibrancy-continued#17] [Q&A]
-  macOS Monterey v12.5.1
- VS Code v1.71.0
- Vibrancy Continued v1.1.4

Hello everyone! First off, just wanted to say I love this project and all you guys are doing.
I get an error whenever I try to 'reload vibrancy' and can't get Vibrancy to work. This pops up in VS Code whenever I run it:
```
Command 'Reload Vibrancy' resulted in an error (A system error occurred (ENOENT: no such file or directory, stat '/home/berge/.vscode-server/bin/784b0177c56c607789f9638da7b6bf3230d47a8c/out/vs/code/electron-sandbox/workbench/workbench.html'))
```
I have uninstalled and reinstalled VS Code as well as Vibrancy and am still getting the same error. Any ideas what this could be?

**Comment by illixion** [comment]
Hi, this error seems to suggest that you’re using VS Code server, are you using VS Code through a web browser? If so, it’s impossible to add transparency and you’ll need to use the VS Code desktop app.

**Reply by michaelberge8** [reply]
I'm using VS Code with the 'Remote - SSH' extension. When I first installed Vibrancy it worked fine both with and without this extension but one time I just opened it up and it wasn't working and I got this error.

**Reply by illixion** [reply]
I think Remote - SSH installs extensions on the server by default, though there should be a way to install extensions locally. Maybe disconnect from the server, install + activate Vibrancy, then re-enable the SSH connection.

**Reply by michaelberge8** [reply]
I uninstalled and reinstalled Vibrancy and disconnected from Remote - SSH and that worked. Thanks for your help for this quick fix, I support everything you're doing this is a great project!

**Reply by michaelberge8** [reply]
I do have one question though: Does the vibrancy effect only work in the current window? I notice when I have two windows side-by-side only one has the blur effect? Is there a way to get a constant blur effect at all times?

**Reply by illixion** [reply]
That might be an Electron restriction, I’m not sure.
